### DISCUSSION TITLE
Audiogridder in Ardour 8 Ubuntustudio 24.04

### DISCUSSION THREAD

**Discussion by eufex** [apohl79/audiogridder#1394] [General]
Has anyone got the plugin working in this setup? I had to do a system rebuild and just getting things working again. Pre-built Audiogridder fails Ardour’s plugin scan, it will just sit there forever unless you tell it to skip. I’ve downloaded Reaper to give it a go a Audiogridder works in there. Would there be any mileage in building from source?

**Comment by matthewhm** [comment]
I have the same issue using Ardour 8.4.0~ds1
"Music for the Marble Palace"
(rev 8.4.0~ds1-2ubuntu8)
Intel 64-bit

on Ubuntu 24.04.01 

The Ardour plugin manager says (for the .so files)

[Info]: Scanning: /usr/lib/vst/AudioGridder.so
/usr/local/share/audiogridder/crashpad_handler: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
Scan was cancelled.

and for the vst3 files:
Invalid VST3 Module Path: ''

even though it's been set to scan /usr/lib/vst3/ where the VST3 plugins are

Any help would be appreciated.

**Reply by eufex** [reply]
I built it a few days ago in Debian 12. Haven’t tried it out yet. If it works I’ll let folks know.

**Reply by eufex** [reply]
Right. After a faff I can confirm that the Linux plugin built in Debian 12 works in AVLinux MX with servers both installed on OSX and Windows. I do have an Ubuntu install but haven’t had time to check it yet. I had some issues and thought it was the network but it was the servers, which need to be the latest build to work with the newly built plugin, no need to build the servers, the ones on the straight download from the AG website work.
